### PR TITLE
<DOC> Virtual world parameter documentation

### PR DESCRIPTION
**Pull request by SEAJang**
# Pull Request Key Information

## Reason for change

Documentation needs to be updated for V4 changes.

## What was changed

Updated the User manual

## Intended Purpose

Provide documentation for the Virtual World identifying parameters

## Testing Completed

No code testing was done as only the documentation was updated. 
Made certain all links in the User Manual related to Virtual world parameters work.

### CODE REVIEW

File: USER_MANUAL.md [hunk 1]
```diff
@@ -28,6 +28,33 @@ Email: <sally@highwoodemissions.com>
   - [5. Simulation Settings](#5-simulation-settings)
     - [\<parameter\_level\> (simulation\_settings)](#parameter_level-simulation_settings)
     - [\<version\> (simulation\_settings)](#version-simulation_settings)
+    - [\<start\_date\>](#start_date)
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:31]: Updates to the TOC should go in a separate commit or PR

File: USER_MANUAL.md [hunk 2]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+**Notes of caution:** The site types defined in this file must correspond to the values in the [sites_file](#sites_file). Refer to the [Virtual World Defining Files](#10-virtual-world-defining-files) for more details.
+
+#### &lt;equipment_group_file&gt;
+
+**Data Type:** String
+
+**Default input:** None
+
+**Description:** This parameter is defined by a string specifying the name of the CSV file containing data on the equipments in the simulation. It's an _optional_ file designed to refine site characteristics and reduce the need for redundant definitions of equipment groups.
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:448]: What do you mean by reduce need for redundant definitions of equipment groups?

File: USER_MANUAL.md [hunk 3]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+**Notes of caution:** The component and sources must correspond to the values found in the [equipment_group_file](#equipment_group_file) and [emissions_file](#emissions_file). Refer to the [Virtual World Defining Files](#10-virtual-world-defining-files) for more details.
+
+### &lt;site_samples&gt;
+
+**Data Type:** Integer
+
+**Default input:** None
+
+**Description:** This variable is a integer indicating the number of sites to subset.
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:472]: This description needs more clarity

File: USER_MANUAL.md [hunk 4]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+**Data type:** Boolean
+
+**Default input:** False
+
+**Description:** Specify if the weather envelopes will be considered when determining the different methods work hours per each day.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Even if weather is considered false, [weather_file](#weather_file) must be present and valid.
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:488]: Is this still true?

File: USER_MANUAL.md [hunk 5]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+**Data Type:** String
+
+**Default input:** None
+
+**Description:** This parameter is specifies the name of the csv file that is used to describe the emissions rate characteristics. Refer to the section [Virtual World Defining Files](#10-virtual-world-defining-files) for comprehensive instructions on setting up this file.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Column headers in the file must correspond to an [emission rate source(ERS)](#ers), [non-repairable emission rate source(NR_ERS)](#nr_ers), or their equivalent when provided in a more granular setting.
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:542]: I know what you mean by this, but is there a better way to word it? I think this will be confusing to anyone but us.

File: USER_MANUAL.md [hunk 6]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+#### &lt;ERS&gt;
+
+**Data Type:** String
+
+**Default input:** None
+
+**Description:** The name of a column in the [emissions file](#emissions_file) that is used to parameterize the emissions characteristics of all the general repairable emissions. This parameter will be overwritten by the equivalent value, when provided at a more granular scale.
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:550]: This one is better, but same feedback as above - Perhaps we need a better way to describe it here than at a more granular scale.

File: USER_MANUAL.md [hunk 7]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+**Notes of caution:**  Available techniques for estimating LPR make a number of problematic assumptions. Ultimately, we have relatively poor data on LPR and the relationship between LPR and NRd. Modeling results are extremely sensitive to LPR. Given that LPR is elusive, we strongly recommend that a broad range of LPR values is evaluated in LDAR-Sim before any decisions are made. For more information, refer to discussions in the main text and supplementary information of Fox et al. (2021).
+
+When the parameter [multi](#multi) is set to false, it will affect the emission production rate observed in the simulation. In this scenario, new emissions won't be generated if there's already an existing emission for the specified emission source.
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:568]: Specified emission source might be confusing since at this level we don't specify it

File: USER_MANUAL.md [hunk 8]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+**Notes of caution:** The column headers are case sensitive.
+
+#### &lt;LPR&gt;
+
+**Data Type:** Float (Numeric)
+
+**Default input:** None
+
+**Description:**  A numeric scalar representing the Leak Production Rate (LPR), which denotes the emission production rate for repairable emissions. New repairable emissions are generated using a site-level empirical LPR. LPR is the probability that a new repairable emission will arise, each day, for each site.  The LPR encapsulates various factors contributing to emission occurrences, such as facility age, management practices, predictive maintenance, and random chance. By setting the LPR in the virtual world parameter file, a uniform LPR is applied across all facility types, production types, facility ages, etc., unless specified otherwise at a more detailed level. For an extended discussion on LPR, see Fox et al. (2021).
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:562]: Again see my previous comment about how we describe propagating parameters.

Comment (ThomasGalesloot) [reply USER_MANUAL.md:562]: That being said this is probably the clearest explanation.

File: USER_MANUAL.md [hunk 9]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+When the parameter [multi](#multi) is set to false, it will affect the emission production rate observed in the simulation. In this scenario, new emissions won't be generated if there's already an existing emission for the specified emission source.
+
+#### &lt;NRd&gt;
+
+**Data Type:** Integer (Numeric)
+
+**Default input:** 365
+
+**Description:** The natural repair duration or day(NRd) of each repairable emission in number of days. Represents emission removal from the repairable emission pool due to routine maintenance, refits, retrofits, and other unintentional emission repairs.
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:576]: No mention here of NRd being a propagating parameter

File: USER_MANUAL.md [hunk 10]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+The NRd value should be the same for **all** programs
+
+#### &lt;multi&gt;
+
+**Data Type:** Boolean
+
+**Default input:** True
+
+**Description:** Specifies whether a repairable emission source can generate multiple emissions simultaneously. For instance, a flare that's unlit can't produce additional unlit emissions simultaneously.
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:590]: Description does not clearly communicate that this level sets the property for all sources.

File: USER_MANUAL.md [hunk 11]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+**Notes on acquisition:** Users are encouraged to undertake an exercise to assess whether it's logical for a particular source to generate multiple emissions simultaneously.
+
+**Notes of caution:** When set to false, it's possible to observe lower emission production rates than what has been parameterized, as existing emissions inhibit the generation of new emissions.
+
+#### &lt;NR_ERS&gt;
+
+**Data Type:** String
+
+**Default input:** The name of a column in the [emissions file](#emissions_file) that is used to parameterize the emissions characteristics of all the general non-repairable emissions. This parameter will be overwritten by the equivalent value, when provided at a more granular scale.
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:600]: See my other comments about clarity with propagating parameters - Will need to update all propagating emissions parameters to be inline with each other.

File: USER_MANUAL.md [hunk 12]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+**Notes of caution:** When the parameter [NR_multi](#nr_multi) is set to false, it will affect the non-repairable emission production rate observed in the simulation. In this scenario, new emissions won't be generated if there's already an existing emission for the specified emission source.
+
+#### &lt;duration&gt;
+
+**Data Type:** Integer (Numeric)
+
+**Default input:** 365
+
+**Description:** The duration or number of days of each of the non-repairable emission lasts. Represents emission removal from the non-repairable emission pool.
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:624]: More description is needed here

File: USER_MANUAL.md [hunk 13]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+**Default input:** False
+
+**Description:** Specifies whether a non-repairable emission source can generate multiple emissions simultaneously. For instance, a flare that's unlit can't produce additional unlit emissions simultaneously.
+
+**Notes on acquisition:** `NR_multi` serves to act as the counter part of [multi](#multi) for non-repairable emissions.
+
+**Notes of caution:** When set to false, it's possible to observe lower emission production rates than what has been parameterized, as existing emissions inhibit the generation of new emissions.
+
+### &lt;Repairs&gt;
```
Comment (ThomasGalesloot) [inline USER_MANUAL.md:642]: Once again with the repair parameters - Need to talk about them being propagating

**Review by ThomasGalesloot — CHANGES_REQUESTED** [review]
Great work! Generally I think most of my comments were around how we describe the propagating parameters. My suggestion is to give them a tag/grouping that makes it clear they are propagating parameters and describe in general how that works and then attach the flowchart we made to show that.

File: USER_MANUAL.md (hunk 4 above)
Comment (SEAJang) [reply USER_MANUAL.md:488]: Yes. This is on the list of TODO, to make sure that if weather file is not used, that it shouldn't be a required input.

File: USER_MANUAL.md [hunk 14]
```diff
@@ -336,6 +363,345 @@ _TODO_
...
+
+See [weather_readme](LDAR_Sim/src/weather/weather_readme.md) documentation for further details regarding weather in LDAR-Sim.
+
+**Notes of caution:**
+
+Weather file sizes can become quite large, especially when spatial and temporal resolution increase (maximum resolutions of 1.25 degrees and 1 hour, respectively). Modelers must decide how to navigate these tradeoffs, and understand the implications of the resolutions chosen.
+
+If using different weather files for different programs (e.g., when comparing different regions), weather data must be downloaded manually and saved to the inputs folder before beginning simulations, as the automatic downloader built into LDAR-Sim will only download one file at a time.
```
[the remark this answers is not in the document]
Comment (SEAJang) [reply USER_MANUAL.md:630]: Not anymore. Virtual world is now consistent amongst all programs.

**Review by ThomasGalesloot — APPROVED** [review]
